src/components/app/TourForm.tsx

```diff
@@ -0,0 +1,56 @@
+import {Tour} from "../../types/tour";
+import {Button, Grid, TextField} from "@mui/material";
+import React, {ChangeEvent, useState} from "react";
+import {useRouter} from "next/router";
+
+type formProps = {
+    tour: Tour
+}
+
+export default function TourForm({tour}: formProps) {
+    const router = useRouter();
+
+    const saveTour = async (event: any) => {
+        event.preventDefault()
+        console.log('saveTour:', currentTour)
+        //todo: call service to save tour
+    }
+
+    const cancel = () => router.push('/tours')
+
+    const [currentTour, updateValue] = useState(tour)
```

Comment:
I think the convention is to use `useState` with having it as `[value, setValue]` so it is clear throughout the code that we're using state when calling it.